pom.xml

```diff
@@ -133,6 +133,7 @@
         <module>presto-proxy</module>
```

Comment:
Is it possible to have a druid docker image for unit tests? Then we could have something like for SqlServer https://github.com/prestosql/presto/pull/503

Reply:
Or start it in-process since Druid is in Java.

Reply:
Whatever is easier to setup and maintain.

Reply:
This was my attempt to launch druid jvm setup - https://gist.github.com/puneetjaiswal/e3b6aaf3cee9fdc4eefb57633d98edfb but does not seem easy to run and manage. I will try launching it using the DockerContainer.